### ISSUE TITLE
PulseVolume widget breaks on latest update

### ISSUE THREAD

**Issue by Graeme22** [qtile/qtile#4788]
### Issue description

Hello! I recently updated my Arch system. There were a couple big changes: kernel 6.8.8, and Python 3.12. Anyways, somehow one of these things broke the PulseVolume widget, which was working fine before. I looked at the package `python-pulsectl-asyncio` which I installed via the AUR and it seems to be up-to-date, having been updated yesterday. Any ideas what might be going on?

### Version

0.25.1.dev0+g005da458.d20240413; however, my pacman package is qtile 0.25.0-2. Seems weird!

### Backend

X11 (default)

### Config

```python
widget.PulseVolume(background=colors[1], fmt=' Volume: {} ', volume_app='pactl', update_interval=0.2, check_mute_command='pactl get-sink-mute 0', get_volume_command='pactl get-sink-volume 0')
```

### Logs

```text
2024-04-29 11:35:21,956 ERROR libqtile utils.py:import_class():L251 Unmet dependencies for 'libqtile.widget.pulse_volume.PulseVolume':
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/utils.py", line 248, in import_class
    module = importlib.import_module(module_path, __package__)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/lib/python3.12/site-packages/libqtile/widget/pulse_volume.py", line 23, in <module>
    import pulsectl_asyncio
  File "/usr/lib/python3.12/site-packages/pulsectl_asyncio/__init__.py", line 1, in <module>
    from .pulsectl_async import PulseAsync
  File "/usr/lib/python3.12/site-packages/pulsectl_asyncio/pulsectl_async.py", line 19, in <module>
    from .pa_asyncio_mainloop import PythonMainLoop
  File "/usr/lib/python3.12/site-packages/pulsectl_asyncio/pa_asyncio_mainloop.py", line 23, in <module>
    from pulsectl._pulsectl import PA_MAINLOOP_API
ModuleNotFoundError: No module named 'pulsectl'
```

### Required

- [X] I have searched past issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by elParaguayo** [comment]
`python-pulsectl` and `python-pulsectl-asyncio` are not in the main Arch repos. Arch recently published the python 3.12 update as long with updates for packages in the repo. Anything installed previously from the AUR will still only be installed for python 3.11 so you need to reinstall the packages from the AUR.

**Comment by Graeme22** [comment]
Reinstalling didn't do anything at first, but I cleared the paru cache and reinstalled again which did the trick.
